<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="hbLayout2">
      <item>
       <widget class="QLabel" name="statisticslbl_main">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statisticslbl_second">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="statistSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="vSpacer5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widLayout1">
       <layout class="QVBoxLayout" name="vbLayout2">
        <item>
         <layout class="QGridLayout" name="gLayout1">
          <item row="0" column="0">
           <widget class="QPushButton" name="btn1">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn2">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn3">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn4">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btn5">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btn6">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btn7">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn8">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn9">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widLayout2">
       <layout class="QVBoxLayout" name="vbLayout3">
        <item>
         <spacer name="vSpacer4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpt">
          <property name="minimumSize">
           <size>
            <width>29</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>29</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="vSpacer3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>158</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widLayout3">
       <layout class="QGridLayout" name="gLayout3">
        <item row="1" column="0">
         <spacer name="hSpacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="vSpacer2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="gLayout2">
          <item row="1" column="0">
           <widget class="QGroupBox" name="gb2">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="vbLayout4">
             <item>
              <widget class="QCheckBox" name="cb">
               <property name="text">
                <string>show statistics</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="gb1">
            <property name="title">
             <string>Player</string>
            </property>
            <layout class="QGridLayout" name="gLayout4">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="vbLayout1">
               <item>
                <widget class="QRadioButton" name="rbtn1">
                 <property name="text">
                  <string>play with computer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbtn2">
                 <property name="text">
                  <string>play for two</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="vSpacer1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="hbLayout1">
            <item>
             <widget class="QPushButton" name="btnOk">
              <property name="text">
               <string>Ok</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancel">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="vSpacer6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
